pkg/vcs/git.go

```diff
@@ -234,19 +234,19 @@ func gitParseCommit(output, user, domain []byte, ignoreCC map[string]bool) (*Com
 			break
 		}
 	}
-	sortedCC := make([]string, 0, len(cc))
+	sortedCC := make(Recipients, 0, len(cc))
```

Comment:
Perhaps these variables with "cc" in their name should be renamed to "recipients"?